### PR TITLE
Fix `post` values in result

### PR DESCRIPTION
**Pull request by vivekjain10**
Fixes result considering example value of `6` for `method_code:http_errors:rate5m{method="post", code="500"}`

### CODE REVIEW

File: content/docs/querying/operators.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ would have been no match as the metrics do not share the same set of labels.
 The entries with methods `put` and `del` have no match and will not show up in the result:
 
     {method="get"}  0.04            //  24 / 600
-    {method="post"} 0.1             //  12 / 120
+    {method="post"} 0.1             //   6 / 120
```
Comment (brian-brazil) [inline content/docs/querying/operators.md]: This should be .05

File: content/docs/querying/operators.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ left:
 
     {method="get", code="500"}  0.04            //  24 / 600
     {method="get", code="404"}  0.05            //  30 / 600
-    {method="post", code="500"} 0.1             //  12 / 120
+    {method="post", code="500"} 0.1             //   6 / 120
```
Comment (beorn7) [inline content/docs/querying/operators.md]: And here 0.05, too.

**Comment by vivekjain10** [comment]
Oh yes. Fixed!

**Comment by beorn7** [comment]
:+1: Thanks.

**Pull request merged**